table/queries/derived_satellite_scans.sql

```diff
@@ -129,7 +129,15 @@ CREATE TEMP FUNCTION OutcomeString(domain_name STRING,
                 WHEN (SELECT LOGICAL_OR(answer.matches_control.asn)
                       FROM UNNEST(answers) answer)
                       THEN "✅answer:matches_asn"
-                ELSE CONCAT("❓answer:not_validated:", AnswersSignature(answers))
+                # TODO: delete, this case is covered by all the cert cases above
+                WHEN (SELECT LOGICAL_OR(NOT (a.https_response_status IS NULL))
+                      FROM UNNEST(answers) a)
+                      THEN CONCAT("❗️answer:unvalidated_https_connection:", AnswersSignature(answers))
+                WHEN (SELECT LOGICAL_OR(NOT (a.http_response_status IS NULL))
```

Comment:
I also suggest we drop the HTTP test. It makes it harder to reason about the methodology by involving another port and another fetch.